lib/vdsm/storage/lvm.py

```diff
@@ -1277,12 +1277,12 @@ def _lvs_proc_info(vg, lvs):
 #
 
 
-def getPV(pvName):
+def getPV(pv_name):
     """
     Return PV named tuple. Raise se.InaccessiblePhysDev if the
     PV does not exist.
     """
-    return _lvminfo.getPv(_fqpvname(pvName))
+    return _lvminfo.getPv(_fqpvname(pv_name))
```

Comment:
The issue already exists, but we should really not use the same name for different things.

The function argument may be a wwid (e.g. xxxyyy). _fqpvname(pv_name) returns a path
(e.g. /dev/mapper/xxxyyy). We need to use consistent names for different things in the entire code base.

It is possible that getPV() works both with path and wwid - this is another issue.
I think the right way to handle this is to accept only path - same value used by lvm when creating pvs.

If the system uses wwids to access pvs in some place, the caller need to create path from the wwid. This layer should not know anything about wwids, only paths.